extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -195,7 +211,7 @@ function _checkIfCanAddMeasurementsToDisplaySet(srDisplaySet, newDisplaySet, dat
     return;
   }
 
-  if (!newDisplaySet instanceof ImageSet) {
+  if ((!newDisplaySet) instanceof ImageSet) {
```

Comment:
Shouldn't this be 

`if (((newDisplaySet instanceof ImageSet)) {`

Reply:
This was changed by my linter automatically, I'll just undo the change and leave it as it was.